lib/src/main/java/uk/gov/di/ipv/cri/address/library/service/AddressService.java

```diff
@@ -96,6 +102,22 @@ public AddressItem saveAddresses(
         return addressItem;
     }
 
+    private CanonicalAddress normalizePostCodeAddresses(CanonicalAddress address) {
+        String rawPostcode = address.getPostalCode();
+        if (rawPostcode != null && "GB".equalsIgnoreCase(address.getAddressCountry())) {
```

Comment:
This is interesting, we're going to need to update the condition when we make this change https://govukverify.atlassian.net/browse/OJ-3105. I wonder if we want something like a `isUKAddress` helper that can be reused and easily updated in one place so that we don't miss things like this?

Reply:
I have included the regression element on **AC 4** I added a CountryCode util class this also moving the logic to one location. I believe with this changes this should be missed by anyone who picks this up